samples/scala-eventsourced-customer-registry/build.sbt

```diff
@@ -10,6 +10,8 @@ enablePlugins(AkkaserverlessPlugin, JavaAppPackaging, DockerPlugin)
 dockerBaseImage := "docker.io/library/adoptopenjdk:11-jre-hotspot"
 dockerUsername := sys.props.get("docker.username")
 dockerRepository := sys.props.get("docker.registry")
+// two Main files in this project changes entry point
+dockerEntrypoint := Seq("bin/main")
```

Comment:
does that second Main really belong in this sample? OK I guess :)

Reply:
I think it is used for the docs somewhere, we have a "snippets" project now but then we'd need to copy some components to show registration with properties there as well.